packages/mobile/src/account/BankAccounts.tsx

```diff
@@ -43,6 +51,7 @@ function BankAccounts({ navigation, route }: Props) {
   const [isOptionsVisible, setIsOptionsVisible] = useState(false)
   const [selectedBankId, setSelectedBankId] = useState(0)
   const walletAddress = useSelector(walletAddressSelector)
+  const [isLoadingBankAccounts, setIsLoadingBankAccounts] = useState(false)
```

Comment:
Check out the [react-async-hook docs ](https://www.npmjs.com/package/react-async-hook). It comes with loading built in and would probably be a lot cleaner than adding in new state

Reply:
oh did not know about this, good suggestion